### ISSUE TITLE
About the PCK accuracy of current model

### ISSUE THREAD

**Issue by tjsmartlzh** [krrish94/CarKeypoints#10]
Hello!
I have noticed that you have trained on your in-house dataset. So could you provide the PCK accuracy of the pre-trained model on your dataset or other datasets you used for train or validate?
Thank you very much!

**Comment by krrish94** [comment]
Thanks for your interest @tjsmartlzh !

As can be seen in section IV-B of our IROS (2018) paper (https://arxiv.org/pdf/1803.02057.pdf Page 7, column 2), we get a PCK of 96.89% on our validation set (at a threshold of 0.1).

Hope this clarifies!
